### ISSUE TITLE
supports jinaai/jina-embeddings-v2-base-code

### ISSUE THREAD

**Issue by wsxiaoys** [iamlemec/bert.cpp#8]
I seems to get jina embeddings convert successfully with

```bash
python bert_cpp/convert.py jinaai/jina-embeddings-v2-base-code models/jina-f16.gguf
```

Seems the only change to original bert is `ALiBi`, as described in https://huggingface.co/jinaai/jina-embeddings-v2-base-code

It'll be nice if we could adapt `ggml_alibi` into this repo for jina embedding support

**Comment by iamlemec** [comment]
Yeah, this would be rad. It does seem like it comes down to adding one or two lines with `ggml_alibi`. The only concern right now is that we're not using a KV cache, so going up to `L=8192` is probably going to be infeasible due to memory. But I do plan on adding in KV cache soon. Let me see if I can get it working in the meantime.
